### ISSUE TITLE
dom alias still not recognized in 0.1.45

### ISSUE THREAD

**Issue by rogergl** [cursiveclojure/cursive#729]
om.dom is still marked as not used.
```Clojure
(ns client.core
  (:require [om.core :as om]
            [om.dom :as dom]
            [cognitect.transit :as t]))

(enable-console-print!)

(def app-state (atom {:distance 70
                      :x 0
                      :text "Hello world!"}))

(def socket (.connect js/io "http://localhost:3000"))

(defn listen [topic]
  (let [r (t/reader :json)
        k (keyword topic)]
    (.on socket topic (fn [data]
                      (swap! app-state assoc k (k (t/read r data)))))))

(listen "x")
(listen "distance")

(defn component [app _ {:keys [dir left top size]}]
  (reify om/IRender
    (render [_]
      (let [dist (:distance app)
            idist (- 245 dist)
            x (if (= dir :left) (:x app) (* -1 (:x app)))]
        (dom/h1 #js{:style #js{:transform (str "rotate(" x "deg)")
                               :position "absolute"
                               :top top
                               :left left
                               :width size
                               :textAlign "center"
                               :verticalAlign "middle"
                               :lineHeight size
                               :color (str "rgb(" idist "," idist "," dist ")")
                               :background (str "rgb(" dist ",0," dist ")")}} (:text app))))))

(om/root
  (fn [app _]
    (reify om/IRender
      (render [_]
        (let [config [{:dir :left :left "30%" :top "22%" :size "14em"}
                      {:dir :right :left "6%" :top "2%" :size "7em"}
                      {:dir :right :left "72%" :top "2%" :size "7em"}
                      {:dir :right :left "6%" :top "64%" :size "7em"}
                      {:dir :right :left "72%" :top "64%" :size "7em"}]]
          (apply dom/div nil (map (fn [c] (om/build component app {:opts c})) config))))))
  app-state
  {:target (. js/document (getElementById "app"))})
```

**Comment by cursiveclojure** [comment]
Thanks for the reproduction case, I'll check this tomorrow.

**Comment by tillda** [comment]
+1 I can confirm this.

Couldn't it be the case that if `bar` in `foo/bar` is generated by macro and therefore not recognised, then the engine thinks that `foo` is not used because it doesn’t have at least one recognised usage?

**Comment by cursiveclojure** [comment]
Yes, that's the case in ClojureScript (see [here](https://github.com/cursiveclojure/cursive/issues/638#issuecomment-73203655)) but I fixed those macros in `0.1.45`. I may have missed a case though, I'll check.

**Comment by cursiveclojure** [comment]
Ok, sorry about this - I fixed Om, then broke it when fixing om-tools, and my tests weren't very good. This is fixed for the next drop.
